### DISCUSSION TITLE
Better Admin-UI for JSON field

### DISCUSSION THREAD

**Discussion by noor-codes** [keystonejs/keystone#7282] [Feature requests]
### Problem
- I am building the Admin-UI for my team who aren't technical people.
- The senior devs asked me to store the information (key-value) in JSON.
- The way currently the JSON field is setup in the Admin-UI, It is plain JSON.

### Solution
- I need a configuration in `json()` field where I can take the key as an input and the value as an input and then store the result in the database as json just like it does right now. 

The only key difference is that the key's and values will be provided as input which makes things a whole lot easier for the non-technical staff I am working with. 

![Screen Capture_select-area_20220214122748](https://user-images.githubusercontent.com/28804764/153824357-e4fe4905-4df2-448d-b914-42f578191662.png)

**Comment by dcousens** [comment]
Hi @noor-codes,

Thanks for the feedback!
We are actively working on themes related to this, this quarter.
We'll ensure to update this discussion as soon as when we have something to share!
